ams/optimizer/pom.xml

```diff
@@ -262,4 +270,46 @@
             </plugin>
         </plugins>
     </build>
-</project>
\ No newline at end of file
+
+    <profiles>
+        <profile>
+            <id>flink1.14</id>
```

Comment:
I suggest emphasizing the Flink version used by the optimizer in the profile name, for example, it can be changed to `optimizerFlink1.14`

Reply:
I had change to   `optimizer.flink.1.1x `